scripts/gWMS-CMSRunAnalysis.sh

```diff
@@ -6,15 +6,16 @@
 # difficult-to-impossible to run.
 #
 
-# On some sites we know there was some problems with environment cleaning
-# with using 'env -i'. To overcome this issue, whenever we start a job, we have
-# to save full current environment into file, and whenever it is needed we can load
-# it. Be aware, that there are some read-only variables, like: BASHOPTS, BASH_VERSINFO,
-# EUID, PPID, SHELLOPTS, UID, etc.
-set > startup_environment.sh
-sed -e 's/^/export /' startup_environment.sh > tmp_env.sh
-mv tmp_env.sh startup_environment.sh
-export JOBSTARTDIR=$PWD
+# import some auxiliary functions from a script that is intented to be shared
+# with WMCore
+source ./env-CMSRunAnalysis.sh
```

Comment:
this line should go after line 13 (Startup environment)

Reply:
eventually, I agree. I think i moved it three times myself. I will fix it